src/app/shell/personal-cabinet/provider/provider-org-info/provider-org-info.component.html

```diff
@@ -10,25 +10,25 @@
               <mat-icon style="color:#3849F9;">edit</mat-icon>Редагувати
             </button></a>
           <h3>Форма властності</h3>
-          <p>{{provider.ownership}}</p>
+          <p> {{getOwnerShipType((provider$ | async)?.ownership)}} </p>
```

Comment:
it may cause a performance issue, try to create a local var from provider$ and use here

Reply:
<-ng-container *ngIf="provider$ | async as provider">
   <-p> {{ provider.foo}}<-/p>
   <-p> {{ provider.bar}}<-/p>
<-/ng-container>